src/EditUserAccountAccessCheck.php

```diff
@@ -26,10 +26,11 @@ class EditUserAccountAccessCheck implements AccessInterface {
    */
   public function access(AccountInterface $user, AccountInterface $account) {
     $roles = $account->getRoles();
-    foreach ($roles as $role) {
-      if (strpos($role, 'virtual_y_') !== FALSE || $role == 'virtual_y') {
-        return AccessResult::forbidden()->cachePerUser();
-      }
+    $vy_roles = array_filter($roles, function ($role) {
+      return strpos($role, 'virtual_y_') !== FALSE || $role == 'virtual_y';
+    });
```

Comment:
why do we need to modify this place?

Reply:
Was found that the user with admin/editor/whatever role can't edit user account if he has also `virtual_y` role